src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.Linux.cs

```diff
@@ -462,7 +452,14 @@ private void InitializeSysFS()
                     return;
                 }
 
-                _sysFSDriver = new SysFsDriver();
+                try
+                {
+                    _sysFSDriver = new LibGpiodDriver(0);
```

Comment:
>_sysFSDriver [](start = 20, length = 12)

NIT: We should rename _sysFSDriver to instead be something like: _interruptDriver or something since it will be confusing if a libgpioddriver variable is calld sysfs

Reply:
True. Renamed all sysfs variables to interrupt